tools/ci_connector_ops/ci_connector_ops/acceptance_test_config_checks.py

```diff
@@ -28,7 +28,7 @@ def find_connectors_with_bad_strictness_level() -> List[utils.Connector]:
         List[utils.Connector]: List of changed connector that are not matching test strictness level expectations.
     """
     connectors_with_bad_strictness_level = []
-    changed_connector = utils.get_changed_connectors()
+    changed_connector = utils.get_changed_connectors(destination=False, third_party=False)
```

Comment:
@alafanechere So the reason Ive set these flags to false is that this is the current behavior.

But! I dont think the current behavior is true.

What do you thing about changing this so that we run
1. acceptance test config checks
2. allowed_hosts_checks

on both sources and destinations

```suggestion
    changed_connector = utils.get_changed_connectors()
```

Reply:
Ah ok. How about the `allowed_hosts` check? Is that source only?